app/javascript/packages/document-capture/context/failed-capture-attempts.tsx

```diff
@@ -143,10 +145,12 @@ function FailedCaptureAttemptsContextProvider({
     incrementFailedCameraPermissionAttempts();
   }
 
-  const forceNativeCamera =
+  const hasExhaustedAttempts =
```

Comment:
Nice variable naming here. 👍🏻

Reply:
Thank you! Credit goes to @aduth for thinking of this name 😁